deployments/charts/kuma/templates/cp-deployment.yaml

```diff
@@ -124,6 +127,11 @@ spec:
             {{ .Values.controlPlane.resources.limits | toYaml | nindent 14 }}
             {{- end }}
           volumeMounts:
+            {{- if (hasKey .Values.controlPlane "automountServiceAccountToken") }} {{- if (eq .Values.controlPlane.automountServiceAccountToken false) }}
```

Comment:
Would become: `{{- if (not .Values.controlPlane.automountServiceAccountToken) }}`